src/UpcomingBooking/UpcomingBooking.js

```diff
@@ -22,6 +22,19 @@ import Box from '../common/Box';
 import Accordion from './Accordion';
 import type { UpcomingBooking_booking as BookingType } from './__generated__/UpcomingBooking_booking.graphql';
 
+const statusMap = {
```

Comment:
can you make it exportable? so it can be reused, we may need it also for [this design](https://app.zeplin.io/project/5a8e8992554d385c76ad5aeb/screen/5aaac0e515d5bbe68d4337ea)

Reply:
Sure! Done